packages/cli/src/builder/cargo.rs

```diff
@@ -99,9 +101,12 @@ impl BuildRequest {
 
     pub(crate) async fn build(
         &self,
-        mut progress: UnboundedSender<UpdateBuildProgress>,
+        mut progress: UnboundedSender<BuildProgressUpdate>,
     ) -> Result<BuildResult> {
-        tracing::info!("🚅 Running build [Desktop] command...");
+        tracing::info!(
+            dx_src = ?MessageSource::Build,
+            "🚅 Running build [Desktop] command..."
```

Comment:
`[Desktop]` in `🚅 Running build [Desktop] command...` is currently hard-coded.
```suggestion
            "🚅 Running build [{}] command...", self.target_platform)
```